Ugly_Number/Nth_Ugly_Number.kt

```diff
@@ -0,0 +1,52 @@
+import java.util.*
+
+//Checking whether number is Ugly Number or not
```

Comment:
A space after //

Reply:
Sorry, But I mistakenly create a new branch for these changes.